### PR TITLE
Add inUpstreamBuiltParentNode option default value for all kinds of equipments fetches

### PR DESCRIPTION
**Pull request by niconoir**

### CODE REVIEW

File: src/main/java/org/gridsuite/study/server/service/StudyService.java [hunk 1]
```diff
@@ -608,8 +608,12 @@ public String getLineMapData(UUID studyUuid, UUID nodeUuid, String lineId, boole
                 "lines", lineId);
     }
 
-    public String getTwoWindingsTransformersMapData(UUID studyUuid, UUID nodeUuid, List<String> substationsIds) {
-        return networkMapService.getEquipmentsMapData(networkStoreService.getNetworkUuid(studyUuid), networkModificationTreeService.getVariantId(nodeUuid),
+    public String getTwoWindingsTransformersMapData(UUID studyUuid, UUID nodeUuid, List<String> substationsIds, boolean inUpstreamBuiltParentNode) {
+        UUID nodeUuidToSearchIn = nodeUuid;
```
Comment (etiennehomer) [inline src/main/java/org/gridsuite/study/server/service/StudyService.java:612]: UUID nodeUuidToSearchIn = nodeUuid;
        if (inUpstreamBuiltParentNode) {
            nodeUuidToSearchIn = networkModificationTreeService.doGetLastParentNodeBuilt(nodeUuid);
        }

This code could be mutualized nope ?

Comment (SlimaneAmar) [reply src/main/java/org/gridsuite/study/server/service/StudyService.java:612]: Yes
method `getNodeUuidFrom(inUpstreamBuiltParentNode)` ?

File: src/main/java/org/gridsuite/study/server/service/StudyService.java [hunk 2]
```diff
@@ -608,8 +608,12 @@ public String getLineMapData(UUID studyUuid, UUID nodeUuid, String lineId, boole
                 "lines", lineId);
     }
 
-    public String getTwoWindingsTransformersMapData(UUID studyUuid, UUID nodeUuid, List<String> substationsIds) {
-        return networkMapService.getEquipmentsMapData(networkStoreService.getNetworkUuid(studyUuid), networkModificationTreeService.getVariantId(nodeUuid),
+    public String getTwoWindingsTransformersMapData(UUID studyUuid, UUID nodeUuid, List<String> substationsIds, boolean inUpstreamBuiltParentNode) {
```
Comment (SlimaneAmar) [inline src/main/java/org/gridsuite/study/server/service/StudyService.java:611]: Typo `inUpstreamBuiltParentNode` ?

Comment (niconoir) [reply src/main/java/org/gridsuite/study/server/service/StudyService.java:614]: No

File: src/main/java/org/gridsuite/study/server/service/StudyService.java (hunk 1 above)
Comment (niconoir) [reply src/main/java/org/gridsuite/study/server/service/StudyService.java]: Created a method getNodeUuidToSearchIn

**Review by etiennehomer — APPROVED** [review]

**Review by SlimaneAmar — APPROVED** [review]

**Pull request merged**